pylint/checkers/imports.py

```diff
@@ -490,6 +490,8 @@ def _record_import(self, node, importedmodnode):
                 importedname = node.modname
             else:
                 importedname = node.names[0][0].split('.')[0]
+        if node.as_string().startswith('from .'):
```

Comment:
FYI I added the following [comments](https://github.com/vauxoo-dev/pylint/blob/a0f064ec3c4c87832382eeaf06a015cb90f140cf/pylint/checkers/imports.py#L495-L501)
```python
# We need the impotedname with first point to detect local package
# Example of node:
#  'from .my_package1 import MyClass1'
#  the output should be '.my_package1' instead of 'my_package1'
# Example of node:
#  'from . import my_package2'
#  the output should be '.my_package2' instead of '{pyfile}'
```